airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/source.py

```diff
@@ -56,43 +60,49 @@ class ConnectorConfig(BaseModel):
     class Config:
         title = "Source Facebook Marketing"
 
-    account_id: str = Field(description="The Facebook Ad account ID to use when pulling data from the Facebook Marketing API.")
-
-    access_token: str = Field(
-        description='The value of the access token generated. See the <a href="https://docs.airbyte.io/integrations/sources/facebook-marketing">docs</a> for more information',
-        airbyte_secret=True,
-    )
-
     start_date: datetime = Field(
+        title="Start Date",
         description="The date from which you'd like to replicate data for AdCreatives and AdInsights APIs, in the format YYYY-MM-DDT00:00:00Z. All data generated after this date will be replicated.",
         pattern="^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z$",
         examples=["2017-01-25T00:00:00Z"],
+        order=0,
     )
 
     end_date: Optional[datetime] = Field(
+        title="End Date (Optional)",
         description="The date until which you'd like to replicate data for AdCreatives and AdInsights APIs, in the format YYYY-MM-DDT00:00:00Z. All data generated between start_date and this date will be replicated. Not setting this option will result in always syncing the latest data.",
         pattern="^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z$",
         examples=["2017-01-26T00:00:00Z"],
         default_factory=pendulum.now,
+        order=1,
+    )
+
+    account_id: str = Field(
+        title="Account ID",
+        description="The Facebook Ad account ID to use when pulling data from the Facebook Marketing API.",
+        examples=["111111111111111"],
+        order=2,
+    )
+
+    access_token: str = Field(
+        title="Access Token",
+        description='The value of the access token generated. See the <a href="https://docs.airbyte.io/integrations/sources/facebook-marketing">docs</a> for more information',
+        airbyte_secret=True,
+        order=3,
     )
 
-    include_deleted: bool = Field(default=False, description="Include data from deleted campaigns, ads, and adsets.")
+    include_deleted: bool = Field(
+        title="Include Deleted", default=False, description="Include data from deleted campaigns, ads, and adsets.", order=4
+    )
 
     insights_lookback_window: int = Field(
-        default=28,
-        description="The attribution window for the actions",
-        minimum=0,
-        maximum=28,
+        title="Insights Lookback Window", default=28, description="The attribution window for the actions", minimum=0, maximum=28, order=5
```

Comment:
this formatting looks awful, where is this coming from?

Reply:
Corrected.

Reply:
@bazarnov thanks, 
I mean no offense, just was curious how is it possible that the build passed successfully and the code doesn't look formatted with black (usually black produces code with good readability). Is this some IDE tool?